### ISSUE TITLE
FAA Changes to ASWS

### ISSUE THREAD

**Issue by cgulliver** [jakekara/faa-airport-status.py#1]
Looks like this endpoint is deprecated and now uses https://soa.smext.faa.gov/asws/api/airport/status/. The script runs fine with the new endpoint but per the doc, the number of airports in scope is drastically reduced... https://app.swaggerhub.com/apis/FAA/ASWS/1.1.0#/

Can anyone confirm?

**Comment by kimyad812** [comment]
You are right about the new endpoint. The number of airports hasn't been reduced though. The API still includes all the airports but doesn't list them all in the "Available airports include the following:..."  

Hope this answers your question.
